api/Client.json

```diff
@@ -48,6 +48,52 @@
           "deprecated": false
         }
       },
+      "frameType": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Client/frameType",
+          "support": {
+            "webview_android": {
+              "version_added": false
```

Comment:
This should be 43.

Reply:
The Android section on the frameType page specifies not supported. However Chrome for android says version 43.
Do you still want me to go ahead and make the change? 
I believe that Chrome for Android has its own entry in the __compat object

Reply:
@gsouquet: @jpmedley is the ultimate reference for Chrome/Webview Android/Chrome for Android.